src/audio-worklet/free-queue/examples/hello-webassembly/example.c

```diff
@@ -0,0 +1,87 @@
+#include <emscripten.h>
+
+ 
+// Include FreeQueue interface library and dr_mp3 library (For loading and
+// decodin mp3 file).
+
+#define FREE_QUEUE_IMPL
+#include "../../src/interface/free_queue.h"
+#define DR_MP3_IMPLEMENTATION
+#include "dr_mp3.h"
+
+struct FreeQueue *queue;
+drmp3 mp3;
+
+// Function to get FreeQueue address in memory from JavaScript.
+EMSCRIPTEN_KEEPALIVE void *getFreeQueue()
+{
+    return queue;
+}
+
+bool lastFramePushed = true;
+drmp3_uint64 framesRead = 0;
+
+#define FRAME_SIZE 4096
+
+// buffer to store interleaved PCM frames from mp3 
+float buffer[2*FRAME_SIZE];    // 2 channels so 2 * FRAME_SIZE
+// input buffer to store decoded mp3 channel data.
+float* input[] = {
+    (float[FRAME_SIZE]){},
+    (float[FRAME_SIZE]){}
+};
+
+/**
+ * Function to read a FRAME from mp3 and push it to queue.
```

Comment:
Use `//` comment instead of `/** * */`.

Perhaps:
Reads frames from a MP3 file and pushes them to a queue.